test/README.md

```diff
@@ -70,11 +70,16 @@ Tests on the host will run with `runc` as the default runtime.
 However you can select other OCI compatible runtimes by setting
 the `RUNTIME` environment variable.
 
-For example one could use the [Clear Containers](https://github.com/clearcontainers/runtime)
-runtime instead of `runc`:
+For example, to use [crun](https://github.com/containers/crun) instead of `runc`:
+```
+make CONTAINER_DEFAULT_RUNTIME=crun localintegration
+```
+
+If you'd like to run the tests with a runtime of a different type, you need to also specify `$RUNTIME_TYPE`
 
+For example, to use [kata](https://github.com/kata-containers/runtime) with shim v2:
 ```
-make localintegration RUNTIME=cc-runtime
+make CONTAINER_DEFAULT_RUNTIME=kata RUNTIME_TYPE=vm localintegration
```

Comment:
The kata shim v2 binary has a slightly cumbersome name ;-)
```suggestion
make CONTAINER_DEFAULT_RUNTIME=containerd-shim-kata-v2 RUNTIME_TYPE=vm localintegration
```
Not all tests will work with it (at least for now), but nonetheless we support specifying the RUNTIME_TYPE, so would be good to have this documented.